org.eclipse.scout.rt.platform/src/main/java/org/eclipse/scout/rt/platform/opentelemetry/OpenTelemetryTraceIdContextValueProvider.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2010, 2025 BSI Business Systems Integration AG
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.scout.rt.platform.opentelemetry;
+
+import org.eclipse.scout.rt.platform.ApplicationScoped;
+import org.eclipse.scout.rt.platform.config.CONFIG;
+import org.eclipse.scout.rt.platform.logger.DiagnosticContextValueProcessor.IDiagnosticContextValueProvider;
+import org.eclipse.scout.rt.platform.opentelemetry.OpenTelemetryProperties.OpenTelemetryTracingEnabledProperty;
+
+import io.opentelemetry.api.trace.Span;
+
+@ApplicationScoped
+public class OpenTelemetryTraceIdContextValueProvider implements IDiagnosticContextValueProvider {
```

Comment:
Should we add javadoc here? Some other `IDiagnosticContextValueProvider` seem to provide a javadoc.